spec/amp-story-player.md

```diff
@@ -218,6 +218,28 @@ Will cause a custom event to be fired, see `page-attachment-open` and `page-atta
 player.getStoryState('page-attachment');
 ```
 
+## Autoplay
+
+You can opt-out of the default autoplaying experience by using the configuration below. This will prevent the first story in the player to start playing until you call `play()` on the player.
```

Comment:
I think this could be clearer if we further defined what the default behavior is, and clarified that this is a way to opt-out.
random example: "By default, the first Story will automatically start playing when the Player gets visible in the user's viewport. You can opt-out and control blablalba using `play()` (link)"